my-app/src/Board.js

```diff
@@ -0,0 +1,88 @@
+import React, { useReducer } from 'react';
+import logo from './logo.svg';
+import Square from './Square';
+import {
+  BoardSquareContext,
+  BoardSetSquareContext,
+  SquareContext,
+} from './App.js';
+
+// main board component
+function Board() {
+  const boardSquares = React.useContext(BoardSquareContext);
+  const setBoardSquares = React.useContext(BoardSetSquareContext);
```

Comment:
Small thing, but I'd include `useContext` in the import statement like `useReducer`.